index.bs

```diff
@@ -2416,6 +2427,47 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
 the same procedure as other [=assertion signatures=] generated by the [=authenticatorMakeCredential=] operation.
 
 
+## Authenticator taxonomy ## {#sctn-authenticator-taxonomy}
+
+[=Authenticator=] types vary along several different dimensions: [=attachment modality=], employed [[#transport|transport(s)]], whether they
+are [=user verification=]-capable or only [=test of user presence|detect any user's presence=], and whether they support
+[=client-side-resident credential private keys=] or not.
+
+Issue: Expand this section with further discussion about the other dimensions mentioned above.
+
+
+### <dfn>Attachment Modality</dfn> ### {#sctn-authenticator-attachment-modality}
+
+[=Clients=] can communicate with [=authenticators=] using a variety of mechanisms. For example, a [=client=] MAY use a
+platform-specific API to communicate with an [=authenticator=] which is physically bound to a [=client device=]. On the other hand, a
+[=client=] can use a variety of standardized cross-platform transport protocols such as Bluetooth (see [[#transport]]) to discover
+and communicate with [=cross-platform attachment|cross-platform attached=] [=authenticators=]. We refer to [=authenticators=] that
+are part of the [=client device=] as <dfn>platform authenticators</dfn>, while those that are reachable via cross-platform
+transport protocols are referred to as <dfn>roaming authenticators</dfn>.
+
+- A [=platform authenticator=] is attached using a platform-specific transport, called <dfn>platform attachment</dfn>, and is usually not removable from the [=client
+    device=]. A [=public key credential=] bound to a [=platform authenticator=] is called a <dfn>platform credential</dfn>.
+
+- A [=roaming authenticator=] is attached using cross-platform transports, called <dfn>cross-platform attachment</dfn>. Authenticators of this class are removable from, and
+    can "roam" among, [=client devices=]. A [=public key credential=] bound to a [=roaming authenticator=] is called a <dfn>roaming
+    credential</dfn>.
+
+Some [=platform authenticators=] could possibly also act as [=roaming authenticators=] depending on context. For example, a
+[=platform authenticator=] integrated into a mobile device could make itself available as a [=roaming authenticator=] via
+Bluetooth. In this case the [=client=] would identify it only as a [=roaming authenticator=], and not as a [=platform
```

Comment:
suggest:
identify it only -> recognize it

Reply:
Done.